env/stdlib/Std/snippets.yaml

```
@@ -1,5 +1,207 @@
+    - name: succ
+      documentation: Calculate Integer successor.
+    - name: upto 10
+      documentation: Generate a list reperesenting a range.
+    - name: > 10
+      documentation: Compare two Integers. 
+    - name: abs
+      documentation: Calculate absolute value.
+    - name: negate
+      documentation: Calculate Integer negation.
+    - name: pred
+      documentation: Calculate Integer predecessor.
+    - name: < 10
+      documentation: Compare two Integers.
+    - name: == 10
+      documentation: Compare two Integers.
+    - name: miliseconds
+      documentation: TimeInterval in miliseconds.
+    - name: seconds
+      documentation: TimeInterval in seconds.
+    - name: minutes
+      documentation: TimeInterval in minutes.
+    - name: hours
+      documentation: TimeInterval in hours.
+    - name: days
+      documentation: TimeInterval in days.
+    - name: weeks
+      documentation: TimeInterval in weeks.
+  Real:
+    - name: + 3.0
+      documentation: Add two floats.
+    - name: * 2.0
+      documentation: Multiply two floats.
+    - name: - 1.3
+      documentation: Substract float from value.
+    - name: / 2.0
+      documentation: Divide by float value.
+    - name: ^ 2.0
+      documentation: Calculate floating point exponentiation.
+    - name: log 10.0
+      documentation: Calculate floating point logarithm.
+    - name: > 10.0
+      documentation: Compare two floating point numbers. 
+    - name: abs
+      documentation: Calculate absolute value for floating point number.
+    - name: negate 
+      documentation: Calculate negation.
+    - name: floor 
+      documentation: Round a number down to closest Int.
+    - name: ceiling
+      documentation: Round a number up to the closest Int.
+    - name: sin 
+      documentation: Calculate sine.
+    - name: cos
+      documentation: Calculate cosine.
+    - name: exp 
+      documentation: Calculate natural exponent.
+    - name: ln
+      documentation: Calculate natural logarithm.
+    - name: sqrt
+      documentation: Calculate squre root.
+    - name: < 10.0
+      documentation: Compare two floating point numbers.
+    - name: == 10.0
+      documentation: Compare two floating point numbers.
+  Text:
+    - name: + "foo"
+      documentation: Concatenate two texts together.
+    - name: length
+      documentation: Check the text length.
+    - name: startsWith "a"
+      documentation: Check whether the argument is a prefix of given text.
+    - name: endsWith "a"
+      documentation: Check whether the argument is suffix of given text value.
+    - name: characters
+      documentation: Return list of all characters in the text.
+    - name: lowercase
+      documentation: Change all characters to lowercase.
+    - name: reverse
+      documentation: Reverse the text.
+    - name: uppercase
+      documentation: Change all characters to uppercase.
+  Complex:
+    - name: modulus
+      documentation: Calculate modulus.
```

```suggestion
      documentation: Calculate complex modulus.
```

While this is clear in many contexts, there are some where it wouldn't be without the addition.